source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -77,24 +77,46 @@ Http::FilterHeadersStatus RoleBasedAccessControlFilter::decodeHeaders(Http::Head
                 callbacks_->connection()->ssl()->subjectPeerCertificate()
           : "none",
       headers, callbacks_->requestInfo().dynamicMetadata().DebugString());
+  std::string effective_policy_id;
   const absl::optional<Filters::Common::RBAC::RoleBasedAccessControlEngineImpl>& shadow_engine =
       config_->engine(callbacks_->route(), EnforcementMode::Shadow);
   if (shadow_engine.has_value()) {
+    std::string shadow_resp_code = "200";
```

Comment:
Can we switch these to static constants.

Reply:
done.